client/src/components/ImageDropZone.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useDropzone } from "react-dropzone";
+
+export default function ImageDropZone(props) {
+    const [imgs, setImgs] = React.useState([]);
+    const { getRootProps, getInputProps } = useDropzone({
+        accept: "image/*",
+        multiple: false,
+        onDrop: files => {
+            setImgs(
+                files.map(file =>
+                    Object.assign(file, {
+                        preview: URL.createObjectURL(file)
+                    })
+                )
+            );
+            props.returnImgToParent(files[0]);
+        }
+    });
+
+    const previewContainerProps = {
+        width: "30%",
+        height: "auto",
+        marginLeft: "auto",
+        marginRight: "auto"
+    };
+    const previewProps = {
+        width: "100%",
+        height: "auto",
+        display: "block"
+    };
+    const preview = imgs.map((file, i) => (
+        <div style={previewContainerProps} key={i}>
```

Comment:
Ideally we are consistent with style and move those objects into a makeStyles outside of the function and use hooks for styling